openapi/components/schemas/plaid/PlaidLinkTokenResponse.yaml

```diff
@@ -0,0 +1,30 @@
+type: object
+required:
+  - linkToken
+  - expiration
+  - callbackUrl
+properties:
+  linkToken:
+    type: string
+    description: >
+      The Plaid Link token to be used to initialize Plaid Link in your application.
+      This token is single-use and expires after the specified expiration time.
+    example: link-sandbox-12345678-1234-1234-1234-123456789012
+  expiration:
+    type: string
+    format: date-time
+    description: >
+      The ISO 8601 timestamp when this link token expires.
+      Link tokens typically expire after 4 hours.
+    example: "2025-10-05T18:30:00Z"
+  callbackUrl:
```

Comment:
Do we need this if it's defined as part of the schema? It looks like the session id is also included as a path param there anyway.